packages/smart/funcs.go

```
@@ -701,7 +709,7 @@ func ColumnCondition(sc *SmartContract, tableName, name, coltype, permissions, i
 	if isExist {
 		return sc.AccessTable(tblName, `update`)
 	}
-	count, err := model.GetColumnCount(tblName)
+	count, _ := model.GetColumnCount(tblName)
```

ignoring errors are bad